<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>shapeWranglerDialog</class>
 <widget class="QDialog" name="shapeWranglerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>shapeWrangler</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>ADDITIVE SHAPES</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>3</number>
       </property>
       <property name="margin">
        <number>6</number>
       </property>
       <item>
        <widget class="QPushButton" name="createBTN">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>CREATE ADDITIVE</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="title">
          <string>ADDITIVE SHAPES</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>1</number>
          </property>
          <property name="margin">
           <number>3</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="shapeTree">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="headerHidden">
             <bool>true</bool>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteShapeBTN">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>DELETE SELECTED SHAPE</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QCheckBox" name="poseReaderCHK">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>USE POSE READER:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="poseReaderCMB">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="refreshPoseReaderCmbBTN">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>11</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="font">
             <font>
              <pointsize>11</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Add, drive, connect additive blendshape</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>OPTIONS</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>poseReaderCHK</sender>
   <signal>toggled(bool)</signal>
   <receiver>poseReaderCMB</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>491</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>poseReaderCHK</sender>
   <signal>toggled(bool)</signal>
   <receiver>refreshPoseReaderCmbBTN</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>483</y>
    </hint>
    <hint type="destinationlabel">
     <x>333</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
